drivers/SmartThings/zigbee-button/src/test/test_aqara_cube_t1_pro.lua

```diff
@@ -0,0 +1,165 @@
+local test = require "integration_test"
+local cluster_base = require "st.zigbee.cluster_base"
+local t_utils = require "integration_test.utils"
+local zigbee_test_utils = require "integration_test.zigbee_test_utils"
+local capabilities = require "st.capabilities"
+local data_types = require "st.zigbee.data_types"
+local clusters = require "st.zigbee.zcl.clusters"
+local PowerConfiguration = clusters.PowerConfiguration
```

Comment:
unused: this is the luacheck error

Reply:
maybe you should write a configuration test

Reply:
I removed them.